.github/workflows/python-package-pytest.yml

```diff
@@ -10,13 +10,13 @@ on:
     branches: [ main ]
 
 jobs:
-  build:
+  testing:
```

Comment:
What does changing this from `build` to `testing` do? (I checked the link on top but I think this change goes deeper than their README. A quick explanation or link please?)

Reply:
Same as above, in this particular example, the title of the workflow (as it would appear under actions tab) would be "pytest / testing (3.7) (pull_request)" where "pytest" comes from line 4, "testing" from this line (13), 3.7 is part of the version matrix in line 18, and pull_request from line 9.

Before editing the title would be: "Python package test / build (3.7) (pull_request)" (for example, see here: https://github.com/uncscode/particula/runs/4602117804?check_suite_focus=true


More details in general here: https://github.com/uncscode/particula/actions